Paper/README.Rmd

```diff
@@ -54,45 +54,29 @@ The first alternative solution is to include the effects of the data pathology i
 
-Instead of the above alternatives, we advocate an ensemble approach. If the ideal model is a Bayesian mixture model, we can approximate the mixture model with an ensemble of base models fit to different subsets of the input data. The critical assumption here is that the "true" mixture model can be approximated as a convex combination of base models. The goal is to choose a list of base models such that the ideal mixture model is in the span of our list of base models. The way to create this list is by assigning each base model to "cover" a specific subset of the input data. This list of base models can be described as a topological cover of the input space. Note that it is not necessary to specify the exact base models that form the "true" convex combination of base models; it is sufficient to provide a list of models from which the convex combination can be generated. Additionally, it is not necessary that the base models be completely disjoint. However, the coverage of our base models will increase if the base model overlap is minimized.
+Instead of the above alternatives, we advocate an ensemble approach. If the ideal model is a Bayesian mixture model, we can approximate the mixture model with an ensemble of base models fit to different subsets of the input data. The critical assumption here is that the "true" mixture model can be approximated as a convex combination of base models. The goal is to choose a list of base models such that the ideal mixture model is in the span of our list of base models. The way to create this list is by assigning each base model to "cover" a specific subset of the input data. This list of base models is chosen such that their union forms a topological cover of the input space. Note that it is not necessary to specify the exact base models that form the theoretically true convex combination of base models; it is sufficient to provide a list of models from which the convex combination can be generated. Additionally, it is not necessary that the base models be completely disjoint. However, the coverage of the ensemble of base models will increase if the base model overlap is minimized.
```

Comment:
This is where some sort of figure or illustration would be especially helpful, @dgmiller.